pyproject.toml

```diff
@@ -30,6 +30,7 @@ from = "src"
 [tool.poetry.dependencies]
 python = "^3.8.1"
 neo4j = "^5.17.0"
+types-requests = "^2.31.0.20240218"
```

Comment:
I'm curious about why we need this dependency? I removed it while implementing the cohere embeddings/llm due to some conflicts, and I had the impression it was not useful for us?

Reply:
https://github.com/neo4j/neo4j-graphrag-python/pull/142/commits/6844cfbdf30ad865e3816742a31c8ae2a20bb42c